.eslintrc.json

```diff
@@ -1,4 +1,5 @@
 {
+  "root": true,
```

Comment:
@maxpluez Was there another eslint config messing things up? I see in the docs it says this:

> By default, ESLint will look for configuration files in all parent folders up to the root directory. This can be useful if you want all of your projects to follow a certain convention, but can sometimes lead to unexpected results. To limit ESLint to a specific project, place "root": true inside the eslintConfig field of the package.json file or in the .eslintrc.* file at your project's root level. ESLint will stop looking in parent folders once it finds a configuration with "root": true.

Reply:
> @maxpluez Was there another eslint config messing things up? I see in the docs it says this:
> 
> > By default, ESLint will look for configuration files in all parent folders up to the root directory. This can be useful if you want all of your projects to follow a certain convention, but can sometimes lead to unexpected results. To limit ESLint to a specific project, place "root": true inside the eslintConfig field of the package.json file or in the .eslintrc.* file at your project's root level. ESLint will stop looking in parent folders once it finds a configuration with "root": true.

I get this error if I don't add that line:
![Screenshot from 2020-09-10 14-59-03](https://user-images.githubusercontent.com/25847912/92813532-3b038b80-f376-11ea-90bb-9af57bf428f7.png)

Reply:
@maxpluez Do you have different ESLint configs at a root level locally that's causing it? Is that why maybe @rlorenzo saw some linting errors on his end?

Reply:
The purpose of "root": true is to tell eslint that this config file is the top most one at root so that eslint won't go any further to its parents. I don't think this line itself causes any problem. Did you see any problem when testing?

Reply:
@maxpluez Didn't see any issue when testing, was just curious why the upgrade caused this issue

Reply:
Oh. It's just that babel-eslint needs to be updated as well.

Reply:
@maxpluez Ahh.. that makes more sense. So root setting is just needed because you have eslintrc file in your home directory. Doesn't hurt to have this added anyways.